tools/common/lib.go

```diff
@@ -1298,6 +1298,15 @@ func Publish(c *cli.Context, cClient ccli.Client) {
 	close(receiptCh)
 }
 
+type kafkaMessageJSON struct {
+	EnqueueTime string
+	Key         string
```

Comment:
They must be string, or they will be outputted in Base64. I had a version before with []byte, and it is not very useful because of this.